### PR TITLE
Cute Little Statistics

### PR DESCRIPTION
**Pull request by NoctuaNivalis**
Placing some cute little statistics and a wall of shame on the landing page.

- [x] Wall of shame
- [x] Total amount of debts
- [ ] Biggest debt ever/this week
- [x] debt grouped per issuer_type (for example biggest drinker/eater on tap)

TODO:
- [ ] Currently configured (shameful balance) in `config/application.rb`, this should go in a subconfiguration file. [Like here?](http://www.richardhsu.me/posts/2015/04/02/rails-4-custom-configurations.html)

### CODE REVIEW

**Comment by becousae** [comment]
- [ ] debt grouped per issuer_type (for example biggest drinker/eater on tap)

File: config/application.rb [hunk 1]
```diff
@@ -22,5 +22,8 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    # Which is the lowest balance you should be ashamed of.
+    config.shameful_balance = 50 # In eurocents!
```
Comment (Silox) [inline config/application.rb]: I think `config.x.shameful_balance` is a more conventional way to put custom configs. See http://guides.rubyonrails.org/configuring.html#custom-configuration

File: app/controllers/concerns/statistics.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (Silox) [inline app/controllers/concerns/statistics.rb]: Unnecessary newline

File: app/controllers/concerns/statistics.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      hash.merge({10 * group.to_i => count})
+    end
+  end
+
+  private
+
+  def zeus_balance
+    User.zeus.balance
+  end
+
```
Comment (Silox) [inline app/controllers/concerns/statistics.rb]: Unnecessary newline

File: app/controllers/pages_controller.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class PagesController < ApplicationController
+
+  require 'statistics'
+
+  def landing
+    @statistics = Statistics.new
+  end
+
```
Comment (Silox) [inline app/controllers/pages_controller.rb]: Unnecessary newline

File: app/controllers/concerns/statistics.rb (hunk 3 above)
Comment (NoctuaNivalis) [inline app/controllers/concerns/statistics.rb]: I agree on the first line of a file, but this one is required for symmetry with the third line of the file.

File: config/application.rb (hunk 1 above)
Comment (NoctuaNivalis) [inline config/application.rb]: Thanks for linking the correct way of configuring things, I was looking for this, still.

**Pull request merged**